crypto/rsa_extra/rsa_asn1.c

```diff
@@ -153,6 +153,32 @@ int RSA_public_key_to_bytes(uint8_t **out_bytes, size_t *out_len,
 // RSAPrivateKey structure (RFC 3447).
 static const uint64_t kVersionTwoPrime = 0;
 
+// Distinguisher for stripped ACCP RSA private keys, sets zeroed values to NULL
+// because ASN.1 treats absent values as 0, but post-parsing validation logic
+// expects absent values to be NULL. Returns 1 if ACCP stripped private key, 0
+// otherwise.
+static int detect_stripped_accp_private_key(RSA *key) {
```

Comment:
Let's pull `ACCP` out of this. This is a result of how the JCA is defined and ACCP just meets its requirements.

Reply:
sure. will look to you and @torben-hansen for guidance on naming here.

Reply:
@SalusaSecondus -- are OK with referring to "JCA" instead of "ACCP"?

Reply:
Works for me!